float64 closest_point        # minimum range value [m]
float64 farthest_point		 # maximum range value [m]